hoomd_organics/base/simulation.py

```diff
@@ -83,12 +84,30 @@ def __init__(
         self.integrator = None
         self._dt = dt
         self._reference_values = dict()
+        self._reference_values = reference_values
         self._integrate_group = hoomd.filter.All()
         self._wall_forces = dict()
         self._create_state(self.initial_state)
         # Add a gsd and thermo props logger to sim operations
         self._add_hoomd_writers()
 
+    @classmethod
+    def from_system(cls, system, **kwargs):
+        """Initialize a simulation from a system object."""
```

Comment:
We should clarify what a `system object` is here. We could instead say `hoomd_organics.base.System`